### PR TITLE
FOGL-1769 - Change South Systeminfo plugin mode from async to poll

### PR DESCRIPTION
**Pull request by amarendra-dianomic**

### CODE REVIEW

File: VERSION.south.systeminfo [hunk 1]
```diff
@@ -1,2 +1,2 @@
-foglamp_south_systeminfo_version=1.1.0
-foglamp_version>=1.4
+foglamp_south_systeminfo_version=2.0.0
+foglamp_version>=1.3
```
Comment (praveen-garg) [inline VERSION.south.systeminfo]: why 1.3?

File: python/foglamp/plugins/south/systeminfo/systeminfo.py [hunk 2]
```diff
@@ -35,28 +28,12 @@
         'readonly': 'true'
     },
     'assetNamePrefix': {
-        'description': 'Asset name prefix',
+        'description': 'Asset prefix',
         'type': 'string',
         'default': "system/",
-        'order': "1",
-        'displayName': 'Asset Name Prefix'
```
Comment (praveen-garg) [inline python/foglamp/plugins/south/systeminfo/systeminfo.py]: why the displayName is removed?

**Review by praveen-garg — CHANGES_REQUESTED** [review]

**Comment by amarendra-dianomic** [comment]
@praveen-garg changes done. Pl check.

**Review by praveen-garg — APPROVED** [review]

**Review by shariq1501 — APPROVED** [review]
LGTM!

**Pull request merged**